### ISSUE TITLE
Wrong blank space in Terminal when use cmder

### ISSUE THREAD

**Issue by gengxuelei** [Microsoft/vscode#13931]
- VSCode Version:1.6.1
- OS Version:Win10

-Cmder Version:161009a [64 (Stable)

There is a wrong blank space after a commander in Terminal when use cmder's bash but it's ok when use cmd.exe .
There is a waring:'存储不足，无法处理此命令。' when use cmd.exe;
And there has no highlighting in Terminal .
![image](https://cloud.githubusercontent.com/assets/11707936/19473436/3e0e8bc4-955e-11e6-8980-80ac0cd1089b.png)
![image](https://cloud.githubusercontent.com/assets/11707936/19473728/5e0bcdb4-955f-11e6-8a27-7ca767e83252.png)

![image](https://cloud.githubusercontent.com/assets/11707936/19473335/c49ec29a-955d-11e6-912b-06239c206bfe.png)

**Comment by Tyriar** [comment]
This is likely an upstream issue https://github.com/sourcelair/xterm.js, potentially related to https://github.com/Microsoft/vscode/issues/10163

**Comment by gengxuelei** [comment]
This bug has been fixed with vscode-1.11.1 & git-bash "2.12.2.windows.2",settings:
`"terminal.integrated.shell.windows": "C:\\Program Files\\Git\\bin\\bash.exe",`

if create a bashrc with "code ~/.bashrc", and enter under settings to this file, it's fine:
```bash
export LANG=en_US.utf-8
# 设置快速到达路径变量 cd $work
export work="/d/work"
# 设置ls添加颜色提示
alias ls='ls --show-control-chars -F --color'
alias ll='ls --show-control-chars -Falh --color'
# 快速打开资源管理器
alias e.='explorer .'
# wget设置不检查https证书
alias wgets='wget --no-check-certificate'
````
